config/jobs/kubernetes/sig-release/release-branch-jobs/1.26.yaml

```diff
@@ -154,7 +154,7 @@ periodics:
       - --allow-dup
       - --bucket=k8s-release-dev
       - --registry=gcr.io/k8s-staging-ci-images
-      - --extra-version-markers=k8s-stable2
+      - --extra-version-markers=k8s-stable3
```

Comment:
This is not expected, it should still stay at `k8s-stable3`

Reply:
Same for other versions

Reply:
Done for 1.25 and 1.26.